Engine/source/core/util/zip/zipArchive.h

```diff
@@ -235,7 +233,7 @@ class ZipTempStream;
 /// exists and may be released as a resource in the future.
 /// 
 /// To set the password used for zips, you need to modify the #DEFAULT_ZIP_PASSWORD
-/// define in core/zip/zipArchive.h. This password will be used for all zips that
+/// define in torqueConfig.h. This password will be used for all zips that
```

Comment:
Well.. Shouldn't this say "you need to modify the configuration in CMake" or something?